uber/models.py

```diff
@@ -1162,8 +1178,16 @@ def last_first(cls):
         ], else_=func.lower(cls.last_name + ', ' + cls.first_name))
 
     @property
+    def watchlist_guess(self):
+        try:
+            with sa.Session() as session:
+                return session.guess_attendee_watchentry(self).__dict__
```

Comment:
I would recommend replacing ``.__dict__`` with ``.to_dict()``; the latter has a lot of hooks we can use later for things like access control and defining what is included and omitted.  Probably not much of a difference now (though I need to read through the rest of the code to be sure).

```python
return session.guess_attendee_watchentry(self).to_dict()
```